Add Boards Manager installation support for megaTinyCore 1.0.3

per1234 opened the pull request:
Boards Manager URL for testing:
https://raw.githubusercontent.com/per1234/ReleaseScripts/add-103/package_drazzy.com_index.json

per1234 committed:
Add Boards Manager installation support for megaTinyCore 1.0.3

SpenceKonde commented:
Hurray! Thanks! 

Sorry for doing so many releases in rapid succession - I had to rush on 1.0.3 because of the serial problem on the x12/x02 - which I'm planning to start selling on Tindie as an assembled board this week (couldn't sell it without getting serial working imo). Plus a working, tightly integrated neopixel library is a good thing in terms of making these usable for general projects - it hadn't really registered in my mind that we didn't have a working ws2812 library before (and everyone expects a way to write to ws2812's from a microcontroller). I'm wondering if there are any other basic checkboxes I need to fill.

per1234 commented:
Don't worry about it at all on my account. I'm pretty swamped with work lately but it really doesn't take much time for me to update the JSON file, especially with the system you use, so I can always squeeze in something like this. I'm having trouble getting blocks of time for the more involved projects I want to work on, but I still like to find the opportunities to make small contributions to open source projects. Great work on megaTinyCore!

Too often, it seems like open source project maintainers get "release phobia". They put off making releases and it becomes a vicious circle where the number of changes since the last release build up so much that they become more and more afraid to make a release. The result is the users get stuck with the choice of using a release version that is years out of date or a beta test version at some random point in the development process.
